src/extension-manager.js

```diff
@@ -172,7 +173,7 @@ function _initExtension(extensions_name, ofExtensionApi) {
     var extensions;
     return new Promise((resolve, reject) => {
         try {
-            extensions = require(extensions_name);
+            extensions = requireg(extensions_name, true); // Make sure to check for installed extensions globally and skip local packages
```

Comment:
@jvolker Remind me why this was necessary?

Reply:
I can't remember the exact situation causing problems to be honest. It happened to me a lot though when developing extensions on different machines and installing using different sources.

It might have to do with http://docs.openframe.io/dev-env-setup/#clone-and-install-the-repos-to-the-pi or http://docs.openframe.io/dev-env-setup/#npm-link-the-extensions or maybe when starting an extension not published to NPM yet. Honestly, not sure anymore. 

But I'm pretty sure this was important!